ui/frontend/src/components/RouteDetails.vue

```diff
@@ -12,28 +12,48 @@
       >
         Remove
       </button>
+
     </div>
     <div class="card">
       <form @submit.prevent="doSubmit">
         <div class="card-body">
           <PropertyField
             :id="'name'"
-            :label="'Name'"
+            :label="'Route Name'"
             :value="formValues.name"
             :errorMsg="errors['name']"
             :inputHandler="updateField"
             :validator="v(uniqueName)"
           />
-          <PropertyField
-              class="mb-4"
-              :id="'input-files'"
-              :label="'Input Files'"
-              :value="formValues['input-files'] | toString"
-              :inputHandler="updateCollectionField"
-            />
-          <div class="form-group form-input">
-            <label class="form-label" for="input">REGO rule:</label>
+          <h4>Rules</h4>
+          <b-form-group>
+            <small class="form-text text-muted">
+              Select rules for route to filter events
```

Comment:
in my local test, it's not apparing the combo opion
![Screenshot from 2022-07-28 15-34-25](https://user-images.githubusercontent.com/8355621/181612721-92b50062-9252-4044-9fb6-9cf7cb62feab.png)

Reply:
I see - you're probably using the default Postee config which does not use the rules that are for Tracee. It's a good point, I will add the default rules for Postee as part of it.

I've pushed the Tracee specific config, https://github.com/aquasecurity/postee/pull/419/commits/265c2fb81a69beea314e9c423814873cef070434 - you can point Postee to use this config instead of the default using command line options.